charts/posthog/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   sha:
   # -- Posthog image tag, e.g. release-1.31.1
   tag:
-  # -- Default image or tag, e.g. :release-1.31.1
```

Comment:
Why?

Reply:
Should override one of the two above settings instead. Could make that clear in the comment though